### PR TITLE
build: release 2.13.0

### PR DESCRIPTION
**Pull request by ciyer**
The post-sprint release of the UI

/deploy #persist

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changes
 
+## [2.13.0](https://github.com/SwissDataScienceCenter/renku-ui/compare/2.12.0...2.13.0) (2022-11-28)
+
+### Features
+
+* switch to REST endpoint for lineage from KG ([#1857](https://github.com/SwissDataScienceCenter/renku-ui/issues/1857), [#2122](https://github.com/SwissDataScienceCenter/renku-ui/issues/2122))
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.md:7]:
```suggestion
### Bug fixes

* fix error when downloading session logs and improve interface ([#2139](https://github.com/SwissDataScienceCenter/renku-ui/issues/2139), [#2160](https://github.com/SwissDataScienceCenter/renku-ui/issues/2160))

### Features

* switch to REST endpoint for lineage from KG ([#1857](https://github.com/SwissDataScienceCenter/renku-ui/issues/1857), [#2122](https://github.com/SwissDataScienceCenter/renku-ui/issues/2122))
```

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]
Should we mention the logs changes too?
P.S: feel free to re-phrase the text in the suggestion.

File: CHANGELOG.md (hunk 1 above)
Comment (ciyer) [reply CHANGELOG.md:7]: I added a mention of the change to the display of the session logs. See the latest commit.

Comment (lorenzo-cavazzi) [reply CHANGELOG.md:12]: Great, thank you!

**Pull request merged**